rootio/scanner.go

```diff
@@ -64,8 +64,20 @@ func (s *baseScanner) Next() bool {
 		return false
 	}
 	next := s.i < s.n
-	s.cur++
-	s.i++
+	switch t := s.tree.(type) {
```

Comment:
By "the value", do you mean s.tree ? because according to what i tested, it's never of type *ttree nor *tchain. I guess i'm forgetting or misunderstanding something. I'll give it another shot :)

Reply:
Thanks for the hint !